jiant/modules/sentence_encoder.py

```diff
@@ -103,11 +97,8 @@ def forward(self, sent, task, reset=True):
         # Task-specific sentence embeddings (e.g. custom ELMo weights).
         # Skip computing this if it won't be used.
         if self.sep_embs_for_skip:
-            if isinstance(self._text_field_embedder, BertEmbedderModule):
-                task_word_embs_in_context = self._text_field_embedder(
-                    sent, task._classifier_name, is_pair_task=is_pair_task
-                )
-
+            if isinstance(self._text_field_embedder, PytorchTransformersEmbedderModule):
```

Comment:
Is this if clause still necessary? The two branches seem identical.

Reply:
Good point. Fixing.